packages/repo-tools/src/commands/package/schema/openapi/fuzz.ts

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import fs from 'fs-extra';
+import { paths as cliPaths } from '../../../../lib/paths';
+import chalk from 'chalk';
+import { spawn } from '../../../../lib/exec';
+import { getPathToCurrentOpenApiSpec } from '../../../../lib/openapi/helpers';
+import { ConfigSources } from '@backstage/config-loader';
+import YAML from 'js-yaml';
+import { join } from 'path';
+import { OptionValues } from 'commander';
+import { sync as existsSync } from 'command-exists';
+
+async function fuzz(opts: OptionValues) {
+  const resolvedOpenapiPath = await getPathToCurrentOpenApiSpec();
+
+  if (!existsSync('st')) {
+    console.log(
+      chalk.red(
+        `Please install schemathesis globally with 'python -m pip install schemathesis'. Then run this command again.`,
+      ),
+    );
+    process.exit(1);
+  }
+
+  const openapiSpec = YAML.load(
+    await fs.readFile(resolvedOpenapiPath, 'utf8'),
+  ) as { info: { title: string } };
+  const configSource = ConfigSources.default({
+    rootDir: cliPaths.targetRoot,
+  });
+  const config = await ConfigSources.toConfig(configSource);
+  const pluginId = openapiSpec.info.title;
+  const args = [];
+  if (opts.debug) {
+    args.push(
+      '--cassette-path',
+      cliPaths.resolveTargetRoot(join('.cassettes', `${pluginId}.yml`)),
+    );
+  }
+
+  if (opts.limit) {
+    args.push('--hypothesis-max-examples', opts.limit);
+  }
+  args.push('--workers', opts.workers);
+
+  if (opts.useGuest) {
+    // TODO: @sennyeya This should leverage the `guest-provider` if available.
+    args.push('--header', `Authorization: Basic guest`);
```

Comment:
Thinking we might have add a few hardcoded tokens that work in development, similar to the mock service